kettle/update.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
Please address or at least find someone else on our team who's willing to support this bash file :see_no_evil: